docs/api/manager-api/getting-started.md.blade.php

```diff
@@ -0,0 +1,110 @@
+---
+title: Getting Started
+---
+
+# Getting Started
+
+The Manager API allows you to gather common node information, access logs, control running processes and handle ledger snapshots on demand.
+Requests should contain valid JSON RPC 2.0 data which are transformed over HTTP / HTTPS protocol using GET method.
+
+<x-alert type="warning">
+All HTTP requests have to be sent with the `Content-Type: application/vnd.api+json` header. If the header is not present, it will result in malformed responses or request rejections.
+</x-alert>
+
+## Installation
+
+The Manager API comes by default with an installation of core. In case you don't have it installed, you can add it manually by running `yarn global add @arkecosystem/core-manager`.
+
+## Alias
+
+`manager`
+
+## Configuration
+
+The Manager API comes with default configuration. Additionally you can customize API specific settings in .env or app.json file.
+
+<x-alert type="warning">
+We strongly recommend setting basic authentication using Argon2Id hashing method with custom secret and using HTTPS access only.
```

Comment:
We strongly recommend setting basic authentication using the Argon2Id hashing method with a custom secret and using HTTPS access only.